engine/src/main/scala/cromwell/services/MetadataServiceActor.scala

```diff
@@ -64,38 +44,26 @@ object MetadataServiceActor {
 
 case class MetadataServiceActor(serviceConfig: Config, globalConfig: Config) extends Actor {
 
-  private var journal = List.empty[MetadataEvent]
+  private val dataAccess = DataAccess.globalDataAccess
```

Comment:
I can make the change, but the same globalDataAccess is going to be passed in at least for the short term.